docs/release_notes.md

```diff
@@ -1,5 +1,33 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Bug Fixes
+
+- Filtered characters to get valid identifiers in Maya for Linux. *AdonisFX-1033*
+- Fixed element selection check at node creation. *AdonisFX-1031*
+- Fixed error messages not displaying the proper status of the AdnSimshape collider connections. *AdonisFX-1034*
+- Prevent AAD file export if there is nothing to export or the only thing to export failed. *AdonisFX-1032*
+- Fixed manipulation of referenced nodes while using the AdonisFX paint tool. *AdonisFX-1147*
+- Fixed a bug that did not allow to add attachments on a muscle deformer created on referenced geometry. *AdonisFX-506*
+- Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
+- Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialization. *AdonisFX-1165*
+- Fixed paint flood operation not working correctly for multi influence constraints. *AdonisFX-1146*
+
+### Features
```

Comment:
Create a section "Core" where you add:
- attach to geo
- slide on geo
- mass support in muscle solvers

OF course, with a proper description

Reply:
Added "Core" section before the "Maya" section.